settings.gradle.kts

```diff
@@ -28,15 +28,15 @@ pluginManagement {
     }
   }
   plugins {
-    id("com.github.johnrengelman.shadow") version "8.1.1"
+    id("com.gradleup.shadow") version "8.3.0"
     id("com.gradle.develocity") version "3.18.2"
     id("com.gradle.plugin-publish") version "1.1.0"
   }
 }
 
 plugins {
   id("com.gradle.develocity")
-  id("org.gradle.toolchains.foojay-resolver-convention") version("0.8.0")
+  id("org.gradle.toolchains.foojay-resolver-convention") version ("0.8.0")
```

Comment:
nit: this either shouldn't have a space or should remove the `()`

Reply:
Ah, I like it, you pay attention to the details :)
Autoformatting changed it, so I thought I'd add it to the commit.
Reverted the change since it had nothing to do with this PR.
I'll create a new one to remove the (), so it is more consistent with the rest.